waiter/src/waiter/core.clj

```diff
@@ -1076,7 +1087,8 @@
                                               (throw (ex-info (str "authentication must be one of: '"
                                                                    (str/join "', '" (sort authentication-providers)) "'")
                                                               {:authentication authentication :status 400}))))
-                                          (sd/validate service-description-builder service-description {}))))
+                                          (sd/validate service-description-builder service-description
+                                                       {:profile->overrides profile->overrides}))))
```

Comment:
Why can't `service-description-builder` have access to `profile-overrides` in all cases?  (initialize in constructor)

Reply:
let's do it in a separate PR, `service-description-defaults` can also move.